exercises/diffie-hellman/tests/diffie-hellman.rs

```diff
@@ -67,6 +67,69 @@ fn test_secret_key_correct_big_numbers() {
     assert_eq!(secret, expected);
 }
 
+#[test]
+#[ignore]
+fn test_public_key_correct_biggest_numbers() {
+    // two biggest 64bit primes
+    let p: u64 = 0xF_FFF_FFF_FFF_FFF_FC5;
```

Comment:
Just a style thing: with hexadecimal values, it is unconventional to divide the digits into groups of three, as you do here. It is much more common to group the digits to some power of 2, because every two hex digits corresponds to exactly one byte.

```suggestion
    let p: u64 = 0xFFFF_FFFF_FFFF_FFC5;
```

The same goes, of course, for all the other hex literals.

Reply:
Done